SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -227,9 +227,19 @@ void USpatialSender::SendRemoveComponents(Worker_EntityId EntityId, TArray<Worke
 	}
 }
 
+void USpatialSender::CreateServerWorkerEntity()
+{
+	RetryServerWorkerEntityCreation(PackageMap->AllocateEntityId(), 1);
+}
+
 // Creates an entity authoritative on this server worker, ensuring it will be able to receive updates for the GSM.
-void USpatialSender::CreateServerWorkerEntity(int AttemptCounter)
+void USpatialSender::RetryServerWorkerEntityCreation(Worker_EntityId EntityId, int AttemptCounter)
 {
+	if (EntityId == SpatialConstants::INVALID_ENTITY_ID)
+	{
+		EntityId = PackageMap->AllocateEntityId();
```

Comment:
Is this a critical error?

Reply:
oops, that's legacy from become I split out the no args method, nice spot